app/models/user.rb

```diff
@@ -43,7 +43,8 @@ class User < ApplicationRecord
       assignments_email: false,
       recent: true,
       recent_email: false,
-      system_message_email: false
+      system_message_email: false,
+      popover_v2_toggle: true
```

Comment:
First of all, why `v2`? I'd name this attribute in user preferences simply `popovers_enabled`. Secondly, I'd move it onto the first level, as `popovers` are **NOT** belonging under notifications.
E.g.
```ruby
default_settings(
    time_zone: 'UTC',
    notifications_settings: {
      assignments: true,
      assignments_email: false,
      recent: true,
      recent_email: false,
      system_message_email: false
    },
    popovers_enabled: true
  )
```

Reply:
i named them that just because popovers exist already, and so that when debbugging it would be clear these are two seperate implementations, but i aggree popovers_enabled is better